ts/aliases.d.ts

```diff
@@ -421,6 +421,10 @@ declare module DevExpress.common.charts {
     export type StateStoreType = DevExpress.common.grids.StateStoreType;
     /** @deprecated Use DevExpress.common.grids.SummaryType instead */
     export type SummaryType = DevExpress.common.grids.SummaryType;
+    /** @deprecated Use DevExpress.common.charts.ShiftLabelOverlap instead */
+    export type FunnelLabelOverlap = DevExpress.common.charts.ShiftLabelOverlap;
+    /** @deprecated Use DevExpress.common.charts.ShiftLabelOverlap instead */
+    export type PieChartLabelOverlap = DevExpress.common.charts.ShiftLabelOverlap;
```

Comment:
Parent namespace is incorrect:

```suggestion

declare module DevExpress.viz.dxFunnel {
    /** @deprecated Use DevExpress.common.charts.ShiftLabelOverlap instead */
    export type FunnelLabelOverlap = DevExpress.common.charts.ShiftLabelOverlap;
}

declare module DevExpress.viz.dxPieChart {
    /** @deprecated Use DevExpress.common.charts.ShiftLabelOverlap instead */
    export type PieChartLabelOverlap = DevExpress.common.charts.ShiftLabelOverlap;
}
```

Reply:
One more thing: move these lines down under the `// v23.1` header:

```ts
// v23.1

declare module DevExpress.viz.dxFunnel {
...
```

Reply:
Made changes